app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="post-form-cont">
```

Comment:
- [ ] OPTIONAL
There is no corresponding _comments_ view file (_views/comments/new.html.erb_) which is intended to contain the **form user interface**. Kindly ensure to create a view with form in the `views/comments/new.html.erb` file which accepts input for creating _comments_. Kindly note that this step is **required** for you app to function as intended!

Reply:
@samlorlah Sorry I made a mistake  and I thought I saved my comment about this part which was not implemented . Please make sure to implement it as mentioned by the reviewed  because it is required for this project and will make you app fully works . :+1: :